annotations/templates/annotations/list_texts.html

```diff
@@ -67,4 +67,19 @@
 <table class="table">
   <column id='test'/>
 </table>
+<script>
+$("#order_title").click(function(){
+  sessionStorage.setItem('order', 'title');
+});
+$("#order_publication").click(function(){
+  sessionStorage.setItem('order', 'created');
+});
+function getQueryString(q){
+  page = document.getElementById('page_number_'+q);
+  if(sessionStorage.getItem('order') == null)
+    page.href='?page='+q+'&order_by=title';
+  else
+    page.href='?page='+q+'&order_by='+ sessionStorage.getItem('order');
+}
+</script>
```

Comment:
Why are we doing this? Can't we keep track of everything in the GET parameters? What happens when the session ends?